packages/components/src/internal/components/forms/QueryFormInputs.tsx

```diff
@@ -255,6 +255,7 @@ export class QueryFormInputs extends React.Component<QueryFormInputsProps, State
                                         onToggleDisable={this.onToggleDisable}
                                         placeholder="Select or type to search..."
                                         previewOptions={col.previewOptions === true || showQuerySelectPreviewOptions}
+                                        queryFilters={col.lookup.hasQueryFilters() ? List(col.lookup.getQueryFilters()) : undefined}
```

Comment:
I believe this should be respected in `resolveDetailEditRenderer` and `SampleStatusInput` (sad face) as well.